src/test/java/com/marklogic/appdeployer/command/schemas/LoadSchemasDataTest.java

```diff
@@ -0,0 +1,32 @@
+package com.marklogic.appdeployer.command.schemas;
+
+import org.junit.Test;
+
+import com.marklogic.appdeployer.AbstractAppDeployerTest;
+import com.marklogic.appdeployer.command.Command;
+import com.marklogic.appdeployer.command.databases.DeployContentDatabasesCommand;
+import com.marklogic.appdeployer.command.databases.DeploySchemasDatabaseCommand;
+import com.marklogic.appdeployer.command.databases.DeployTriggersDatabaseCommand;
+import com.marklogic.appdeployer.command.restapis.DeployRestApiServersCommand;
+
+public class LoadSchemasDataTest  extends AbstractAppDeployerTest {
+
+	@Test
+	public void testSchemaLoading() {
```

Comment:
I think the test should do 2 more things - verify that the test schema files are now in the application's schemas database (should be able to use appConfig.newSchemasDatabaseClient() for that), and then include an @After method that calls undeploySampleApp() to ensure everything is cleaned up. 

I don't have a Travis/Jenkins/etc job for running these tests, but I do run them before creating new builds, so I put a lot of value in them. They take just under 20 minutes right now to run - a lot of the time is spent deleting app servers and waiting for ML to restart.